CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 [unreleased]
 
+[2.0.0-alpha.12]
```

Comment:
I'd remove this line, sorry for the confusion

Reply:
cool removing, no worries!

Reply:
done @leblowl